### ISSUE TITLE
Code the feature to backspace words

### ISSUE THREAD

**Issue by utsav0** [utsav0/typingFinger#1]
In this project, right now only character backspace is available but we need to add even word backspace.

**Comment by lodenrogue** [comment]
Can you define what you mean by even word backspace?

**Comment by utsav0** [comment]
It's just backspace. 

Basically the thing is,  when we are at a certain character at certain word and we press the backspace key. 

Then the cursor gets one character back but if the cursor is on the exact starting of the word and then we press the backspace key, nothing happens. 

So, what we need is, if the cursor is on the starting of a word, and we press the backspace key then the the cursor should get back to the last character of the previous word.

**Comment by lodenrogue** [comment]
I noticed on monkeytype.com it doesn't go to the end of the previous word after you backspace through the current word. Are you seeing similar behavior in this app?

**Comment by utsav0** [comment]
Yes, actually, I also didn't noticed that they also don't have this feature. 

But, we want to add it anyway.
